presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -918,6 +918,27 @@ public void testInsertIntoPartitionedTable()
         }
     }
 
+    @Test
+    public void testInsertIntoSpecialPartitionName()
+    {
+        // Generate temporary table name.
+        String tmpTableName = generateRandomTableName();
+        // Clean up if temporary table already exists.
+        dropTableIfExists(tmpTableName);
```

Comment:
This is not needed after @xiaoxmeng 's refactor.

Reply:
Is this refactor already in-place?

Reply:
Yes. #19864